ckan/tests/test_coding_standards.py

```diff
@@ -586,12 +586,15 @@ def find_unprefixed_string_literals(filename):
     u'ckanext/datapusher/tests/test_interfaces.py',
     u'ckanext/datastore/controller.py',
     u'ckanext/datastore/db.py',
+    u'ckanext/datastore/backend/postgres.py',
```

Comment:
Again, please do not add new files to the whitelist :wink:

Simply prefix all string literals with either `u` or `b` [as stated by the rules](http://docs.ckan.org/en/latest/contributing/unicode.html).